lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -123,7 +161,7 @@ define([
             var requests = AjaxHelpers.requests(this),
                 teamMemberships = TeamSpecHelpers.createMockTeamMemberships([]),
                 teamsView = createTopicTeamsView({ teamMemberships: teamMemberships });
-            verifyActions(teamsView, {showActions: true});
+            verifyActions({showActions: true});
```

Comment:
Bonus points for updating the `TODO` code.

Reply:
lol, but not enough.  should just be `verifyActions()` at this point :-p